NOTICE-binary

```diff
@@ -63,4 +58,10 @@ Apache Ratis Server API
 Copyright 2017-2020 The Apache Software Foundation
 
 Apache Ratis Thirdparty Miscellaneous
-Copyright 2017-2020 The Apache Software Foundation
\ No newline at end of file
+Copyright 2017-2020 The Apache Software Foundation
+
+Apache Spark
+Copyright 2014 and onwards The Apache Software Foundation
+
+ResolverUtil.java
```

Comment:
> It's strange that we put this file in the binary release kit instead of the source release kit.

Hi @WillemJiang , this part is copied from the NOTICE of log4j-core-2.17.2.jar
```
Apache Log4j Core
Copyright 1999-2012 Apache Software Foundation

This product includes software developed at
The Apache Software Foundation (http://www.apache.org/).

ResolverUtil.java
Copyright 2005-2006 Tim Fennell
```
And since Celeborn's code doesn't depend on the class, we think it should be in the binary release kit instead of the source release kit.

Please correct me if I misunderstood, thanks!

Reply:
OK, thanks for the explanation.